images/icons/src/left-arrow.svg

```diff
@@ -0,0 +1 @@
+<svg width="8" height="11" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M7 1L1.169 5.198c-.225.163-.225.441 0 .604L7 10" stroke="#000"/></svg>
```

Comment:
Newline error here, `svgo` probably stripped it out when I created the file

Reply:
👍